docs/source/modin/supported/general_supported.rst

```diff
@@ -64,6 +64,12 @@ Data manipulations
 |                             |                                 |                                  | performance. You can force the use of the Snowflake|
 |                             |                                 |                                  | parser with ``engine=snowflake``                   |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``read_excel``              | D                               |                                  | Uses local pandas to read excel files, using the   | 
```

Comment:
The definition of `D` above this table says:
```
``D`` stands for defaults to single node pandas execution via UDF/Sproc
```
I think we can just say `Y` here instead - since all the parameters are supported.